FFMPEG_INSTALLATION.rst

```diff
@@ -0,0 +1,11 @@
+=========
+FFMPEG Installation
```

Comment:
can you put this description in a general INSTALLATION.rst file and perhaps link it to the readme.rst file?